src/app/modules/shared_models/section_details.nim

```diff
@@ -5,27 +5,27 @@ import ../../../app_service/service/contacts/dto/contacts
 import ../../../app_service/common/types
 
 QtObject:
-  type ActiveSection* = ref object of QObject
+  type SectionDetails* = ref object of QObject
     item: SectionItem
 
-  proc setup(self: ActiveSection) =
+  proc setup(self: SectionDetails) =
     self.QObject.setup
 
-  proc delete*(self: ActiveSection) =
+  proc delete*(self: SectionDetails) =
     self.QObject.delete
 
-  proc newActiveSection*(): ActiveSection =
+  proc newActiveSection*(): SectionDetails =
     new(result, delete)
     result.setup
 
-  proc membersChanged*(self: ActiveSection) {.signal.}
-  proc bannedMembersChanged*(self: ActiveSection) {.signal.}
-  proc pendingRequestsToJoinChanged*(self: ActiveSection) {.signal.}
-  proc pendingMemberRequestsChanged*(self: ActiveSection) {.signal.}
-  proc declinedMemberRequestsChanged*(self: ActiveSection) {.signal.}
-  proc communityTokensChanged*(self: ActiveSection) {.signal.}
+  proc membersChanged*(self: SectionDetails) {.signal.}
+  proc bannedMembersChanged*(self: SectionDetails) {.signal.}
+  proc pendingRequestsToJoinChanged*(self: SectionDetails) {.signal.}
+  proc pendingMemberRequestsChanged*(self: SectionDetails) {.signal.}
+  proc declinedMemberRequestsChanged*(self: SectionDetails) {.signal.}
+  proc communityTokensChanged*(self: SectionDetails) {.signal.}
 
-  proc setActiveSectionData*(self: ActiveSection, item: SectionItem) =
+  proc setActiveSectionData*(self: SectionDetails, item: SectionItem) =
```

Comment:
```suggestion
  proc setSectionDetailsData*(self: SectionDetails, item: SectionItem) =
```

Reply:
`ChatContentView` will be kept in memory, but `ChatMessagesView` will be unloaded (members, chats list, input field and etc won't). As @osmaczko says - it is the heaviest part. I wonder if with these changes we need to keep in memory 5 ChatMessagesView at all? I guess we can unload them if they are not visible/active (but need to test if it will not impact the performance).

Reply:
@jrainville we still have it, but we only unload the `ChatMessagesView` instead of `ChatContentView`.